src/formatDate.js

```diff
@@ -9,6 +9,64 @@
  */
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const dateSeparated = date.split(fromFormat[3]);
```

Comment:
Consider validating that the 'date' string actually contains the expected separator before attempting to split it. This will make your function more robust and prevent potential runtime errors.